howitworks.md

```diff
@@ -11,7 +11,7 @@ from a Certificate Authority (CA). Let's Encrypt is a CA. In order to get a
 certificate for your website's domain from Let's Encrypt, you have to demonstrate
 control over the domain. With Let's Encrypt, you do this using software that uses
 the [ACME protocol](https://ietf-wg-acme.github.io/acme/), which typically runs
-on your web host.
+on your web host. This software is called the "client".
```

Comment:
Maybe: This software is called the "ACME client" ?

Reply:
I choose to just call it the "client" so there wouldn't be any confusing if further down the "ACME" part was left out. But I was in doubt about it, so I've included it now :slightly_smiling_face: